backend/typescript/services/implementations/campService.ts

```diff
@@ -140,9 +234,17 @@ class CampService implements ICampService {
         // if there are no campers, we return an empty string
         return "";
       }
+
+      const flattenedCampers = campers.map((camper) => {
+        const { formResponses, ...formObj } = camper;
+        return {
+          ...formResponses,
+          ...formObj,
+        };
```

Comment:
is it possible to do this?
```suggestion
        return {
          ...camper.formResponses,
          ...camper.formObj,
        };
```

Reply:
camper does not have "formObj" field,

`const {formResponses, ...formObj} = campers;` // here I am getting formResponses field from camper object, and everything else is encapsulated in "formObj"

does it make sense? did I understand your question correctly?